examples/decoding/decoding_rsa.py

```diff
@@ -0,0 +1,191 @@
+
+[2] Laakso, A. & Cottrell, G.. "Content and cluster analysis:
+ assessing representational similarity in neural systems." Philosophical
+ psychology 13.1 (2000): 47-76.
+
+[3] Kriegeskorte, N., Marieke, M., & Bandettini.  P. "Representational
+ similarity analysis-connecting the branches of systems neuroscience."
+ Frontiers in systems neuroscience 2 (2008): 4.
+
+[4] Cichy, R. M., Pantazis, D., & Oliva, A. "Resolving human object recognition
+in space and time. Nature neuroscience" (2014): 17(3), 455-462
+"""
+
+# Authors: Jean-Remi King <jeanremi.king@gmail.com>
+#          Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+from pandas import read_csv
+import matplotlib.pyplot as plt
+
+from sklearn.model_selection import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import roc_auc_score
+from sklearn.manifold import MDS
+
+import mne
+from mne.io import read_raw_fif, concatenate_raws
+from mne.datasets import visual_92_categories
+
+print(__doc__)
+
+data_path = visual_92_categories.data_path()
+
+# Define stimulus - trigger mapping
+fname = op.join(data_path, 'visual_stimuli.csv')
+conds = read_csv(fname)
+print(conds.head(5))
+
+##############################################################################
+# Let's restrict the number of conditions to speed up computation
+max_trigger = 24
+conds = conds[:max_trigger]  # take only the first 24 rows
+
+##############################################################################
+# Define stimulus - trigger mapping
+conditions = ['/'.join(map(str,
+                           list(c[:2]) + [('not-' if i == 0 else '') +
+                                          conds.columns[k]
+                                          for k, i in enumerate(c[2:], 2)]))
+              for c in conds.values]
+print(conditions[:10])
+
+##############################################################################
+# Let's make the event_id dictionary
+event_id = dict(zip(conditions, map(int, conds.trigger + 1)))
+event_id['0/human bodypart/human/not-face/animal/natural']
+
+##############################################################################
+# Read MEG data
+n_runs = 4  # 4 for full data (use less to speed up computations)
+fname = op.join(data_path, 'sample_subject_%i_tsss_mc.fif')
+raws = [read_raw_fif(fname % block) for block in range(n_runs)]
+raw = concatenate_raws(raws)
+
+events = mne.find_events(raw, min_duration=.002, verbose=True)
+print(events[:10])
+
+events = events[events[:, 2] <= max_trigger]
+mne.viz.plot_events(events, sfreq=raw.info['sfreq'])
+
+##############################################################################
+# Epoch data
+picks = mne.pick_types(raw.info, meg=True)
```

Comment:
Since the classifier deals with a numpy array, this ensures that we don't include trigger channels